bitnami/redis/README.md

```diff
@@ -544,6 +544,7 @@ useExternalDNS:
     ttl: 10
 ```
 
+
```

Comment:
Unrelated change.

Reply:
Yes, I added a new line in redis README.md just to make sure the Redis workflow gets triggered. This will not enter the final round of changes. It is still in testing.

Reply:
`$ git commit --allow-empty -m "Trigger pipeline"`